### PR TITLE
chore: add timecost telemetry for cli preview check

### PR DESCRIPTION
**Pull request by meifans**

### CODE REVIEW

**Comment by kuojianlu** [comment]
How about add `preview-prerequisites-check-start` and `preview-prerequisites-check` events, then add the time cost in the property of `preview-prerequisites-check`?

File: packages/cli/src/cmds/preview/preview.ts [hunk 1]
```diff
@@ -282,72 +288,80 @@ export default class Preview extends YargsCommand {
...
-    // init service log writer
-    this.serviceLogWriter = new ServiceLogWriter();
-    await this.serviceLogWriter.init();
+      // clear background tasks
+      this.backgroundTasks = [];
+      // init service log writer
+      this.serviceLogWriter = new ServiceLogWriter();
+      await this.serviceLogWriter.init();
+
+      /* === start ngrok === */
```
Comment (kuojianlu) [inline packages/cli/src/cmds/preview/preview.ts:329]: start ngrok, prepare dev env is not parts of prerequisites checking, right?

Comment (meifans) [reply packages/cli/src/cmds/preview/preview.ts]: Yes, changed.

File: packages/cli/src/cmds/preview/preview.ts [hunk 2]
```diff
@@ -282,41 +288,55 @@ export default class Preview extends YargsCommand {
...
-    // clear background tasks
-    this.backgroundTasks = [];
-    // init service log writer
-    this.serviceLogWriter = new ServiceLogWriter();
-    await this.serviceLogWriter.init();
+      // clear background tasks
+      this.backgroundTasks = [];
+      // init service log writer
+      this.serviceLogWriter = new ServiceLogWriter();
+      await this.serviceLogWriter.init();
```
Comment (kuojianlu) [inline packages/cli/src/cmds/preview/preview.ts:327]: Move this block outside, just before "start ngrok".

Comment (meifans) [reply packages/cli/src/cmds/preview/preview.ts]: Added.

**Review by kuojianlu — APPROVED** [review]

**Review by Alive-Fish — APPROVED** [review]

**Pull request merged**